docs/usage/type-conversions.md

```diff
@@ -589,23 +697,24 @@ console.log(result); // [1, 6, 7, 14]
 ```
 
 ### Calling Javascript functions from Python
```

Comment:
In terms of sections I find it a bit confusing to have this title here while we had,

 - "PyProxy memory management calling Javascript functions from Python"

just before. Generally maybe it would be better to create a separate page for more advanced memory management  topics that it linked from within the type translations (same as is done for making a release from under contributing)? As otherwise this page is getting both long and dense.

Reply:
> create a separate page for more advanced memory management

This is probably a good idea. I agree that the organization is getting a bit messed up. I have another PR planned to reduce the memory management burden in another way which would also require changes to the docs generally reducing the complexity of memory management in typical use cases.

Anyways my thought was that if/when these PRs are accepted we can go back afterwards and do a global edit on the docs to make it flow reasonably again. At least I don't want to try too hard to make the docs read well if we haven't decided for sure to make the change.